blocks/editable/readme.md

```diff
@@ -0,0 +1,93 @@
+# `Editable`
```

Comment:
Can we get this on the handbook within the "Block API" tree?

Reply:
Done.